content/work/coachhub.md

```diff
@@ -24,10 +24,10 @@ tags:
   - Case Study
 sample:
   desktop:
-    src: orcas/desktop.jpg
+    src: orcas/coachhub-desktop.jpg
```

Comment:
@dvdherron I see these are renamed, I guess because we can't have images with duplicate names? That makes me a bit worried going forward -- it seems like it'd be easy to by mistake add a new image with the same name as an existing image, since we're magically flattening the file structure. Is there a reason we can't maintain the file structure even after image processing, instead of dumping them all in one folder?

I might give that a quick try...

Reply:
@dvdherron Hm, it looks like the 11ty plugin doesn't really support nested output directories. I'll file an issue, but in the meantime I wonder about ditching our custom `filenameFormat` option and just letting the plugin generate a unique hash for each file? That would prevent unintentional duplicate image names, at least for now. What do you think?

cc @mirisuzanne

Reply:
That makes sense to me.

Reply:
Another option would be to store a list of all file names, and emit an error on build if there are duplicates. But I'm not sure that's really better than a hash.

Reply:
@jgerigmeyer Another option could be to add a hash in addition to the filename with [shortHash](https://github.com/11ty/eleventy-img/issues/51#issuecomment-775186353) (inside the options block). Don't know if it's worth installing another package just for that 🤷🏽 

Yes, those images that had the same name were loading incorrectly. Makes sense that they should all be unique or nested.

Reply:
@dvdherron Oh, that's an interesting idea. I don't think we'd have to add a new package -- we could just include the `id` arg in the returned custom filename...